spec/integ/crypto/verification.spec.ts

```diff
@@ -104,180 +108,166 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("verification (%s)", (backend: st
         });
 
         await initCrypto(aliceClient);
+
+        e2eKeyResponder = new E2EKeyResponder(aliceClient.getHomeserverUrl());
+        syncResponder = new SyncResponder(aliceClient.getHomeserverUrl());
+        mockInitialApiRequests(aliceClient.getHomeserverUrl());
+        aliceClient.startClient();
```

Comment:
Isn't `startClient` `async` and shout be awaited here?

Reply:
good spot!

```suggestion
        await aliceClient.startClient();
```